test/testBrowserLanguage.js

```diff
@@ -0,0 +1,139 @@
+import browserLanguage from '../src/nginxconfig/util/browser_language';
+
+class MockLocales{
```

Comment:
```suggestion
class MockLocales {
```
Formatting consistency